src/debugger_requests.jl

```diff
@@ -128,6 +128,15 @@ function set_compiled_functions_modules!(items::Vector{String})
 
     # user wants these compiled:
     for acc in items
+        if acc == "ALL_MODULES_EXCEPT_MAIN"
+            for mod in values(Base.loaded_modules)
```

Comment:
When does this code run? After debugger launches and after the ```my_script.jl``` is loaded? ```my_script.jl``` is the script the user wants to debug

Reply:
This ends up being run on each debugger step.